packages/contracts-bedrock/test/properties/helpers/MockCrossDomainMessenger.t.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { OptimismSuperchainERC20 } from "src/L2/OptimismSuperchainERC20.sol";
+import { SafeCall } from "src/libraries/SafeCall.sol";
+
+contract MockCrossDomainMessenger {
```

Comment:
Is not the cross domain but the L2 <-> Messenger instead. I suggest also changing the name to the file

```suggestion
contract MockL2ToL2Messenger {
```